interfaces/rule_schema.yaml

```diff
@@ -0,0 +1,496 @@
+$id: https://raw.githubusercontent.com/returntocorp/semgrep/develop/semgrep/semgrep/rule_schema.yaml
+$schema: http://json-schema.org/draft-07/schema#
+$comment: '!!if you modify this file, you probably also need to update the one in semgrep-app:server/semgrep_app/rule_schema.yaml!!'
```

Comment:
The semgrep-core rules parser doesn't use this schema right now; we should document that. Also, it might be reasonable to build the semgrep-core rules parser using this schema, which would be better.

Reply:
(I would be happy to do that at some point)